helper_dart/lib/src/facet_list.dart

```diff
@@ -349,3 +361,42 @@ class _FacetList with DisposableMixin implements FacetList {
     _subscriptions.cancel();
   }
 }
+
+extension FilterTracking on FacetList {
```

Comment:
this extension would not be needed, if the adapter already encapsulates attribute + indexName